articles/index.yml

```diff
@@ -1,18 +1,19 @@
 ### YamlMime:Landing
 
 title: Microsoft Quantum Documentation
-summary: Learn how to use the Microsoft Quantum Development Kit and Q#, a language for quantum programming. Learn key concepts and write your first quantum program. Explore the rich tools and libraries of the QDK.
+summary: Explore the concepts of quantum computing and how to use the Microsoft Quantum Development Kit (QDK) and Q#, an open source quantum programming language.
 
 metadata:
   title: Microsoft Quantum Documentation - Q# API Reference
-  description: Learn how to use the Microsoft Quantum Development Kit and Q#, a language for quantum programming. Learn key concepts and write your first quantum program. Explore the rich tools and libraries of the QDK.
+  description: Explore the concepts of quantum computing and how to use the Microsoft Quantum Development Kit (QDK) and Q#, an open source quantum programming language.
```

Comment:
Is this intended to be a repeat of line 4? That feels a bit awkward to me, as the API reference doesn't cover concepts.

Reply:
This is the main landing page for the quantum docs - you can get to both the concepts and the API ref. I clarified the meta title to reflect that.